fixtures/coverall/tests/bindings/test_coverall.kts

```diff
@@ -39,6 +39,34 @@ createSomeDict().use { d ->
     assert(d.coveralls!!.getName() == "some_dict")
 }
 
+createNoneDict().use { d ->
```

Comment:
Where's this from? Seems unrelated to the `bytes` part

Reply:
This test was part of the other language's tests, so I added it to Kotlin as well. Should I move it to a different commit/remove it altogether?

Reply:
Interesting. Yeah, I don't mind having that added, but as a separate commit would be good then.

Reply:
Done.